uktrade/data-hub-frontend
# Replace govuk-react Main component with FE version

## Description of change

As we now have a `<Main>` React component that outputs the correct `<main>` HTML element we should replace all use cases of the govuk-react version.

## Test instructions

1. Edit a One list and everything should look the same
2. Send a referral flow - everything should look the same
3. View the Investment project admin tab - everything should look the same

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
